config/env/development/plugins.js

```diff
@@ -25,7 +25,7 @@ module.exports = ({ env }) => ({
       // tls: { ciphers: 'SSLv3'},
       auth: {
         user: env('SMTP_USER', 'sis.admin@medha.org.in'),
-        pass: env('SMTP_PASSWORD', 'sisadmin123A'),
+        pass: env('SMTP_PASSWORD', 'Admin.SIS407'),
```

Comment:
I would recommend to remove the password from here and make this only:
```suggestion
        user: env('SMTP_USER'),
        pass: env('SMTP_PASSWORD'),
```

So this will require to define both these variables in the `.env` file and we will not reveal the username and password from the code. This is important as the projects are now open source.

Reply:
Yes, this makes sense.